src/components/Learn-Components/Card-Component/learn-card.style.js

```diff
@@ -12,6 +12,10 @@ const CardWrapper = styled.div`
 
     .card-parent{
         border-radius: 2rem;
+        min-height:357px;
+        display:flex;
+        flex-direction:column;
+        justify-content:space-between;
```

Comment:
Is the flex-direction and justify-content really needed?

Reply:
Yes..They are to align the icons to the bottom so that they fall in line.